devbin/build-llvm.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# note: those commands assume a directory structure like:
+
+# ~
+#   src
+#     llvm-project
+#     hail
```

Comment:
Let's promulgate the assumption that `$HAIL` refers to the Hail repository. I think the only option for LLVM is to add a `HAIL_LLVM` env var. You'll want to announce to #team when this merges how to use it.

Reply:
Updated the docs/scripts to use environment variables for the LLVM and Hail repo locations!